admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -332,7 +344,17 @@
         this.dropzone.on(DropzoneEvents.addedFile, (file: PSDropzoneFile) => {
           file.previewElement.dataset.id = file.image_id;
 
-          if (file.is_cover) {
+          // @todo: Improve types int his component (the file type is actually not PSDropzoneFile in all cases).
```

Comment:
left this todo here on purpose, I hope we can clean this up if we have some time later.
But basically during different events this "file" object has different types, so this type of PSDropzoneFile is confusing). And overall types in this component should be reviewed, there is probably more stuff like this